src/main/resources/edu/cornell/kfs/module/purap/document/datadictionary/RequisitionDocument.xml

```diff
@@ -25,6 +25,15 @@
    	<ref bean="RequisitionDocument-vendorEmailAddress"/>
    </list>
    </property>
+   <property name="webScriptFiles" ref="reqsWebScriptFiles" />
```

Comment:
Because the parent bean uses a factory to create the list, I was not able to use <list merge="true"> directly, and instead had to extend the factory and add it to its "sourceList" property.